lib/datadog/tracing/distributed/datadog.rb

```diff
@@ -46,11 +46,18 @@ def inject!(digest, data)
           data[@sampling_priority_key] = digest.trace_sampling_priority.to_s if digest.trace_sampling_priority
           data[@origin_key] = digest.trace_origin.to_s if digest.trace_origin
 
-          build_tags(digest).tap do |tags|
-            inject_tags!(tags, data) unless tags.empty?
+          begin
```

Comment:
Is there any reason for this change?

Reply:
Closing the loop on this: Spoke with Tony and we decided to revert this and update the test instead. The likelihood of tags not being a hash is extremely unlikely, so we shouldn't be generating any exceptions in `build_tags`.